dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs

```diff
@@ -3047,6 +3049,7 @@ public string ExtensionForContentType(string contentType)
 															 new string[] {"mov"    , "video/quicktime"},
 															 new string[] {"qt"		, "video/quicktime"},
 															 new string[] {"avi"    , "video/x-msvideo"},
+															 new string[] {"m4a"	, "audio/x-m4a"},
```

Comment:
Why is it twice? (see line 3029)

Did you mean to add the `m4v` extension?
```suggestion
															 new string[] {"m4v"	, "video/x-m4v"},
```

Reply:
We need the two of them for compatibility right? 
"m4a"<->"audio/mp4"
"m4a"<->"audio/x-m4a"
Remember server-side uses the mapping in both ways.

Reply:
Fixed

Reply:
Thanks!